src/sentry/api/validators/doc_integration.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from jsonschema import Draft7Validator
+from jsonschema.exceptions import best_match
+
+from sentry.utils.json import JSONData
+
+logger = logging.getLogger(__name__)
+
+METADATA_SCHEMA = {
+    "type": "object",
+    "definitions": {
+        "resource": {
+            "type": "object",
+            "properties": {"title": {"type": "string"}, "url": {"type": "string", "format": "uri"}},
+            "required": ["title", "url"],
+            "additionalProperties": False,
+        }
+    },
+    "properties": {"resources": {"type": "array", "items": {"$ref": "#/definitions/resource"}}},
+    "additionalProperties": False,
+}
+
+METADATA_PROPERTIES = list(METADATA_SCHEMA["properties"].keys())
```

Comment:
In fact I'd make a function here:
```py
def get_meta_data_properties(data: Mapping[str, Any]) -> Mapping[str, Any]:
	return { 
	      k: v 
	      for k, v in data.items() 
	      if k in METADATA_PROPERTIES
	}
```

Reply:
theyre slightly different now, i was ensuring only METADATA_PROPERTIES were written to the DB, then also ensuring they were the only ones read, which is redundant. If anything not in METADATA_PROPERTIES is written to that column, we should still return it when serializing the row.